CMakeLists.txt

```diff
@@ -61,11 +61,20 @@ add_executable(CRender src/main.cpp
         src/ui/ui.h
         src/render/entities/components.h
         src/render/entities/registry.cpp
-        src/render/entities/registry.h src/render/draft/draft_renderer.cpp src/render/draft/draft_renderer.h src/render/timer.cpp src/render/timer.h src/util/logger.h src/util/logger.cpp src/util/numbers.h src/render/brdf.h src/util/denoise.h)
+        src/render/entities/registry.h 
+        src/render/draft/draft_renderer.cpp 
+        src/render/draft/draft_renderer.h 
+        src/render/timer.cpp 
+        src/render/timer.h 
+        src/util/logger.h 
+        src/util/logger.cpp 
+        src/util/numbers.h 
+        src/render/brdf.h 
+        src/util/denoise.h)
 
 target_include_directories(CRender PRIVATE src)
 target_include_directories(CRender PRIVATE external)
 
-target_link_libraries(CRender glfw fmt glm embree OpenImageDenoise)
+target_link_libraries(CRender glfw fmt glm embree OpenImageDenoise stdc++fs)
 
-target_compile_definitions(CRender PUBLIC -DGLFW_INCLUDE_NONE -D__STDC_CONSTANT_MACROS)
+target_compile_definitions(CRender PUBLIC -DGLFW_INCLUDE_NONE -D__STDC_CONSTANT_MACROS -DIMGUI_IMPL_OPENGL_LOADER_GLAD)
```

Comment:
No new line EOF

Reply:
So just add in a new line?

Reply:
Yeah

Reply:
Actually nah its fine, its the cmakeliste file